corehq/apps/hqadmin/static/hqadmin/js/project_map.js

```diff
@@ -210,6 +210,18 @@ var projectMapInit = function(mapboxAccessToken) {
         info.update();
     }
 
+    function capitalizeCountryNames(countries) {
+        return countries.map(function(country) {
+            var formattedCountryName = country.charAt(0).toUpperCase();
+            if (country.indexOf(",") > -1) {
```

Comment:
what case is this handling? very nit picky but if we're also cleaning / truncating some names the method name is a little misleading (maybe switch to `formatCountryNames`?)